recipes/eo-learn-geometry/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "eo-learn-geometry" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d7590ff2928e151eaa5ce457c37f9faf8a8101df4ff02ec46e386107a4357ffb
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win and vc<14 or py<35]
+  
+requirements:
+  host:
+    - pip
+    - python
+    - pillow
```

Comment:
Is pillow really needed at install time?

Reply:
probably not, left over from testing. removed in latest commit